stress_test_model_cb.R

```diff
@@ -21,8 +21,6 @@ function_paths <- c(
   file.path(
     "R",
     c(
-      "annual_pd_change_company_technology.R",
```

Comment:
just out of interest, do u know how functions like annual_pd_change_company_sector.R are made available now since u only removed the old file names from the sourced functions but did not add the new ones?

Reply:
they are sourced in the QA scripts.